src/Controller/AuthController.php

```diff
@@ -7,47 +7,47 @@
 
 namespace Lopi\Bundle\PusherBundle\Controller;
 
+use Lopi\Bundle\PusherBundle\Authenticator\ChannelAuthenticatorInterface;
 use Lopi\Bundle\PusherBundle\Authenticator\ChannelAuthenticatorPresenceInterface;
-use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\Security\Core\Exception\AccessDeniedException;
 
 /**
  * AuthController
  *
  * @author Richard Fullmer <richard.fullmer@opensoftdev.com>
  * @author Pierre-Louis Launay <laupi.frpar@gmail.com>
  */
-class AuthController extends Controller
+class AuthController
 {
+    private $pusherConfig;
+    private $authenticator;
+
+    public function __construnct(array $pusherConfig, ?ChannelAuthenticatorInterface $authenticator)
```

Comment:
type in `__construnct` - looks like we should probably add an integration test to make sure this can at least be instantiated